File: Solutions/Java/Left_most_and_right_most_index.java
class Solution {
    
    public pair indexes(long v[], long x)
    {
        // Your code goes here
        int start=-1;
        int end=-1;
        int n=v.length;
        for (int i=0; i<n; i++) {
            if (v[i]==x) {
                start=i;
                break;
            }
        }
        for (int i=n-1; i>=0; i--) {
            if(v[i] ==x){
                end=i;
                break;
            }
        }
        return new pair(start,end);
    }
}

File: Solutions/Python/Left_most_and_right_most_index.py
class Solution:
    def indexes(self, v, x):
        # Your code goes here
        start=-1
        end=-1
        n=len(v)
        for i in range(n):
            if v[i]==x:
                start=i
                break
        for i in range(n-1, -1, -1):
            if v[i] == x:
                end=i
                break
        return [start,end]
